master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Securing access to {{site.prodname}}'s metrics endpoints
+---
+
+## About securing access to {{site.prodname}}'s metrics endpoints
+
+When using {{ site.prodname }} with prometheus metrics enabled, it may be desirable to use network policy
+to limit access to {{ site.prodname }}'s metrics endpoints.
+
+### Purpose of this page
+
+Provide an example workflow for creating network policies to limit access to {{site.prodname}}'s metrics endpoints.
+
+## Prerequisites
+
+- {{site.prodname}} is installed with [prometheus metrics reporting enabled](TODO).
```

Comment:
Do we think it's worth adding a guide on how to enable prometheus metrics? Or should I just link to the relevant reference information?